daprdocs/content/en/getting-started/get-started-component.md

```diff
@@ -55,7 +55,7 @@ spec:
     value: ":"
 ```
 
-You can see that the above file definition has a `type: secretstores.local.file` which tells Dapr to use the local file component as a secret store. The metadata fields provide component specific information needed to work with this component (in this case, the path to the secret store JSON)
+You can see that the above file definition has a `type: secretstores.local.file` which tells Dapr to use the local file component as a secret store. The metadata fields provide component specific information needed to work with this component (in this case, the path to the secret store JSON relative to where you started your app from.)
```

Comment:
I'd clarify this a little more, saying it's where `dapr run` is executed. Also make sure it's aligned with the other reference.